python/spatialdata-js-util/README.md

```
@@ -0,0 +1,335 @@
+
+```bash
+pip install 'spatialdata-js-util[tui]'
+spatialdata-js-util tui
+
+# Or pre-fill the store path so you skip the first prompt
+spatialdata-js-util tui ~/data/xenium.zarr
+```
+
+Pick a command from the home menu and the UI walks you through it:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)